### PR TITLE
Update makefile and workflow

### PR DESCRIPTION
**Pull request by Itai-Nelken**
* Update the makefile:
    - rename the 'test' target to 'build-debug'.
    - add a new 'test' target that test the executable.
* Update the workflow ﻿to use the new targets.

### CODE REVIEW

**Review by CleanMachine1 — APPROVED** [review]

File: makefile [hunk 1]
```diff
@@ -6,5 +8,8 @@ uninstall :
 	rm /usr/bin/\$
 clean :
 	rm temp
+build-debug :
+	gcc dollarskip.c -o temp -Wall
 test :
-	gcc dollarskip.c -o temp -Wall
\ No newline at end of file
+	@echo -e "\e[1mIf you see 'It Works!', then the test was succesful!\e[0m"
```
Comment (bl-ue) [inline makefile:14]:
```suggestion
	@echo -e "\e[1mIf you see 'It Works!', then the test was successful!\e[0m"
```

**Review by bl-ue — APPROVED** [review]
One comment :)

File: makefile (hunk 1 above)
Comment (CleanMachine1) [reply makefile:14]: :+1: Good spot!

Comment (Itai-Nelken) [reply makefile]: Fixed, thanks for noticing

**Comment by CleanMachine1** [comment]
I haven't tested this at all however I trust it will work

**Pull request merged**